app-text/apvlv/apvlv-0.4.0.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit xdg cmake
+
+DESCRIPTION="apvlv is a PDF/EPUB Viewer and its behaviour like Vim"
+HOMEPAGE="https://github.com/naihe2010/apvlv/"
+SRC_URI="https://github.com/naihe2010/apvlv/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug djvu"
+
+DEPEND="
+	>=app-text/poppler-0.5.0[cairo,xpdf-headers(+)]
+	dev-libs/glib:2
+	x11-libs/gtk+:3
+	net-libs/webkit-gtk
+	app-text/ebook-tools
+	djvu? ( app-text/djvu:= )
```

Comment:
Looking at https://github.com/naihe2010/apvlv/blob/master/src/CMakeLists.txt you seem to be missing at least freetype as a dep.

Reply:
freetype is already a dependency of app-text/poppler, which is already included. Including freetype might be redundant.

Reply:
We don't depend on indirect deps like that. Imagine a scenario where one day poppler dropped freetype as a dep, or it became a dependency via a use flag. It'd break apvlv. 

So always list direct dependencies.